src/java.base/share/classes/sun/util/cldr/CLDRCalendarDataProviderImpl.java

```diff
@@ -47,8 +45,8 @@
  */
 public class CLDRCalendarDataProviderImpl extends CalendarDataProviderImpl {
 
-    private static Map<String, Integer> firstDay = new ConcurrentHashMap<>();
-    private static Map<String, Integer> minDays = new ConcurrentHashMap<>();
+    private static final Map<String, Integer> firstDays = new ConcurrentHashMap<>();
```

Comment:
I don't think this is correct. "First Day" is the first day of the week and should not be plural.

Reply:
The map contains several entries so I think the plural form is appropriate here.

Reply:
Current naming highlights the difference in values in this maps: `FIRST_DAY_OF_WEEK` vs `MINIMAL_DAYS_IN_FIRST_WEEK`.
I like this approach.

Reply:
Maybe `firstDayMap`/`minDaysMap` would clarify more, but either way is fine with me